.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint
+
+on: push
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Git repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # all history needed for commitlint to work
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Install Node dependencies
+        run: npm ci --no-audit --no-optional
+
+      - name: Run commit linter
+        run: npm run lint:commit
+
+      - name: Run all other linters
+        uses: wearerequired/lint-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          eslint: true
+          eslint_args: '--ignore-path .gitignore --max-warnings 0'
+          eslint_extensions: 'ts'
```

Comment:
What happens if we don't specify extensions?

Will it try to read all extensions that ESLint supports, and isn't that a good thing?

Reply:
It stupidly [defaults to "js" only extension](https://github.com/wearerequired/lint-action/blob/master/action.yml#L110).